packages/paypal-js/types/v6/components/base-component.d.ts

```diff
@@ -0,0 +1,59 @@
+export type OnApproveDataOneTimePayments = {
+    orderId: string;
+    payerId?: string;
+    billingToken?: string;
+};
+
+export type OnCompleteData = {
+    paymentSessionState: "approved" | "cancelled" | "error";
```

Comment:
I think this should be:
```suggestion
    paymentSessionState: "approved" | "canceled" | "error";
```